src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/LimitedUseItem.java

```diff
@@ -149,4 +150,16 @@ private void updateItemLore(ItemStack item, ItemMeta meta, int usesLeft) {
         }
     }
 
+    @Override
+    public boolean canStack(ItemMeta itemMetaOne, ItemMeta itemMetaTwo) {
+        if (Slimefun.getItemDataService().getItemData(itemMetaOne) != Slimefun.getItemDataService().getItemData(itemMetaTwo)) {
```

Comment:
This returns optionals right? You shouldn't use a != here

Reply:
oh, i didn't notice it return an optional(beacuse i change it in github)